src/man/sssd-files.5.xml

```diff
@@ -84,7 +84,13 @@
                 <refentrytitle>sssd.conf</refentrytitle>
                 <manvolnum>5</manvolnum>
             </citerefentry> manual page for details on the configuration
-            of an SSSD domain.
+            of an SSSD domain. But please note that because the purpose of
+            the files provider is to provide the same data as the UNIX
+            files, just through the SSSD interfaces, not all generic domain
+            options are supported. Likewise, some global options, such as
```

Comment:
Overall I understand, what we are trying to explain here. Definitely it is an improvement. Just the sentence "But please ... supported" is too long. Can we split it somehow like:

The purpose of the files provider is to provide (deliver/make available? - just not to have 2xprovide)
the same data as the UNIX files, just through the SSSD interfaces. Therefore not all generic domain
options are supported.

Reply:
Thank you for the review, I pushed a new version, does it read better to you?

Reply:
Perfect